index.js

```diff
@@ -209,7 +226,7 @@ const runOnce = async (
     const mdPaths = glob.sync(`${markdownDirectory}/**/*.md`, {ignore, nodir: true})
     const pumlPaths = glob.sync(`${pumlDirectory}/**/*.puml`, {ignore, nodir: true})
     const pumlLinks = new PumlLinks({
-        pumlPaths, shouldShortenLinks, pumlServerUrl
+        pumlPaths, shouldShortenLinks, pumlServerUrl, polrUrl
```

Comment:
can we change to `shouldUsePolrUrl`?

Reply:
this is actual url to polr url shortener service, so 'polrUrl' name makes more sense - did not change this.